.github/workflows/trivyimagescan.yaml

```diff
@@ -30,7 +35,7 @@ jobs:
             echo "running make build benchmark"
             make build
         - name: Run Trivy vulnerability scanner
-          uses: aquasecurity/trivy-action@7b7aa264d83dc58691451798b4d117d53d21edfe
+          uses: aquasecurity/trivy-action@master
```

Comment:
it'd be better to pin some release version or tag instead of master which may not be stable at all times...